modules/control-utility/src/test/java/org/apache/ignite/util/KillCommandsCommandShTest.java

```diff
@@ -228,10 +231,14 @@ public void testCancelConsistencyTask() throws InterruptedException {
 
         cfg.setName(consistencyCancheName);
         cfg.setBackups(SERVER_NODE_CNT - 1);
+        cfg.setCacheMode(CacheMode.REPLICATED);
```

Comment:
This test passes with partitioned cache too. Why do we need set replicated here?